Doc/library/unittest.rst

```diff
@@ -1830,6 +1830,9 @@ Loading and running tests
       Return a sorted sequence of method names found within *testCaseClass*;
       this should be a subclass of :class:`TestCase`.
 
+      .. deprecated:: 3.10
```

Comment:
No, method `getTestCaseNames` should not be deprecated. Only function `getTestCaseNames` should be deprecated.